test/libero-config/fixtures/configFixtureStandAlone.js

```diff
@@ -1,22 +1,22 @@
 const Color = require('color');
 
-const config = { data: {} };
+const config = {data: {}};
 
-config.data.baselinegrid = {space: {}};
-config.data.baselinegrid.space.extra_small_in_px = 12;
-config.data.baselinegrid.space.small_in_px = '!expression baselinegrid.space.extra_small_in_px * 2';
+config.data.baseline_grid = {space: {}};
```

Comment:
`baselinegrid` to match the actual config. (This will be extracted soon, but should keep things internally consistent anyway.)